### ISSUE TITLE
Composite keys?

### ISSUE THREAD

**Issue by PhillipTommerholt** [IKAMR/KDRS_Metadata#19]
Hi guys,
Nice tool!

The first thing I notice is that it seems that composite keys is not shown in the output?

Best Phillip

**Comment by solfeggietto** [comment]
Please let us know if any SIARD file of any version or produced by any app/version does not get their primary, foreign or candidate keys properly stored in the Excel output file for the individual table tab in the spreadsheet.

The counter of keys present as you can see in the app dialog window at completed job and in the logfile, may have bugs as this is a feature added late in the latest version(s) and need be tested and possibly done bugfix on. I added it because those statistics give a good overview of the presence or lacking of the different keys in the tables.

Example from a logfile on KDRS Metadata v0.94:

Tables: 77
schema0: , rows: 373066, Total rows: 373066, Max rows: 124979, PKs: 40, FKs: 1, CKs: 94, noPKs: 37, noFKs: 0, noCKs: 0, yesFKs: 77, yesCKs: 77

**Comment by PhillipTommerholt** [comment]
This metadata.xml has a composite PK on tables "Orders".
[metadata.zip](https://github.com/IKAMR/KDRS_Metadata/files/6107769/metadata.zip)
```
<primaryKey>
            <name>PK_Orders</name>
            <column>OrderID</column>
	<column>CustomerID</column>
```

The output is:
![image](https://user-images.githubusercontent.com/15984159/110456675-235d5380-80ca-11eb-94a5-0a86922f3cde.png)
Only showing one the columns of the PK.

**Comment by solfeggietto** [comment]
We must fix this bug coming week.

**Comment by solfeggietto** [comment]
Solved in version [0.9.5-RC1](https://github.com/IKAMR/KDRS_Metadata/releases/tag/v0.9.5-RC1)
